### DISCUSSION TITLE
LTSP boot error!

### DISCUSSION THREAD

**Discussion by ikki0123** [ltsp/ltsp#728] [Q&A] [answered]
So I'm testing LTSP on virtual machines using Virtual Box.
On the server machine a Debian 11 is installed, I followed the installation steps from here https://ltsp.org/docs/installation/.
All processes worked, except when I went to test the client machine, the error image follows.
I'm new to linux syste
![Capturar](https://user-images.githubusercontent.com/109396275/179328499-aaebb10a-c506-409e-a11a-e43a9965a017.PNG)
m

**Comment by alkisg** [comment, marked as answer]
You have a separate /var partition, you need to tell LTSP about it.
See https://github.com/ltsp/ltsp/discussions/250#discussioncomment-247842
Converting into a discussion...

**Reply by ikki0123** [reply]
Thanks, it worked!!
